src/one_dragon/base/operation/one_dragon_context.py

```
@@ -251,9 +251,16 @@ def switch_instance(self, instance_idx: int) -> None:
         self.one_dragon_config.active_instance(instance_idx)
         self.current_instance_idx = self.one_dragon_config.current_active_instance.idx
         self.reload_instance_config()
-        self.init_controller()
+        self.on_switch_instance()
         self.dispatch_event(ContextInstanceEventEnum.instance_active.value, instance_idx)
 
+    def on_switch_instance(self) -> None:
+        """
+        切换实例后的回调，用于更新 controller 配置
```

Ruff (0.14.10)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)